src/levelset/levelSetSegmentationObject.cpp

```diff
@@ -1796,7 +1797,26 @@ int LevelSetSegmentationBaseObject::evalPart(const std::array<double,3> &point)
  */
 std::array<double,3> LevelSetSegmentationBaseObject::evalNormal(const std::array<double,3> &point, bool signedLevelSet) const
 {
-    return _evalNormal(point, signedLevelSet);
+    // Project the point on the surface and evaluate the point-projection vector
+    std::array<double, 3> projectionPoint;
+    std::array<double, 3> projectionNormal;
+    evalProjection(point, signedLevelSet, &projectionPoint, &projectionNormal);
+
+    BITPIT_UNUSED(projectionPoint);
```

Comment:
There is no need to mark projectionPoint as unused. The macro is only needed to silence compiler workings related to unused function arguments.